sat/cli/status/status_module.py

```diff
@@ -574,5 +571,9 @@ def rows(self):
                 # This finally block always runs regardless of whether the
                 # loop was prematurely `continue`d or not.
                 components.append(component)
+            # Logging missing sessions
+            for session_id, count in self.missing_sessions.items():
+                LOGGER.debug('Session id %s which applies to %d nodes is missing',
+                             session_id, count)
```

Comment:
This is still inside the `for` loop, so it's going to be printed for each component rather than just once after iterating over all the components. Please move it outside the for loop.

Reply:
Addressed